### PR TITLE
Create DIVD-2021-00014.md

### PR DESCRIPTION
**Pull request by cookiemonster**

### CODE REVIEW

File: csirt.divd.nl/cases/DIVD-2021-00014.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+layout: case
+title: DIVD-2021-00014 - Kaseya Unitrends
+divd: DIVD-2021-00014
+short: DIVD recommends not exposing the on-premise Kaseya Unitrends servers.
+author: Victor Gevers
+lead: Victor Gevers
+redirect_from:
+  - /DIVD-2021-00014/
+status : Open
+excerpt: sers of on-premise Kaseya Unitrends are advised to not expose this service directly to the internet
```
Comment (Lennaert89) [inline csirt.divd.nl/cases/DIVD-2021-00014.md:11]: sers?

**Review by Lennaert89 — CHANGES_REQUESTED** [review]
1 spelling error, looks good for everything else. 
Only doing a case file and not a blog post at this time? (blog post means it shows up in our RSS feed)

File: csirt.divd.nl/cases/DIVD-2021-00014.md (hunk 1 above)
Comment (MrSeccubus) [reply csirt.divd.nl/cases/DIVD-2021-00014.md:11]: Users

File: csirt.divd.nl/cases/DIVD-2021-00014.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
+---
+layout: case
+title: DIVD-2021-00014 - Kaseya Unitrends
+divd: DIVD-2021-00014
+short: DIVD recommends not exposing the on-premise Kaseya Unitrends servers.
```
Comment (MrSeccubus) [inline csirt.divd.nl/cases/DIVD-2021-00014.md:5]: Exposing to what?

File: csirt.divd.nl/cases/DIVD-2021-00014.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+status : Open
+excerpt: sers of on-premise Kaseya Unitrends are advised to not expose this service directly to the internet
+researchers:
+- Wietse Boonstra
+- Victor Gevers
+cves: n/a
+product: Kaseya Unitrends
+versions: < 10.5.2
+recommendation: "
+Please do not expose this service or the clients directly to the internet until Kaseya has patched these vulnerabilities.
```
Comment (MrSeccubus) [inline csirt.divd.nl/cases/DIVD-2021-00014.md:19]: Keep it facts and figures. Do not, not please.

File: csirt.divd.nl/cases/DIVD-2021-00014.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+"
+-patch_status:	 	n/a
```
Comment (MrSeccubus) [inline csirt.divd.nl/cases/DIVD-2021-00014.md:21]: This says unpatched, text says patch available.

File: csirt.divd.nl/cases/DIVD-2021-00014.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+
+### English
+
+## Summary
+A DIVD researcher has identified several vulnerabilities in the [Kaseya Unitrends](https://www.unitrends.com/products/enterprise-backup-software) backup
+product version < 10.5.2.
+
+
+## What you can do
+Please do not expose this service or the clients (running default on ports 80, 443, 1743,
```
Comment (MrSeccubus) [inline csirt.divd.nl/cases/DIVD-2021-00014.md:34]: Again, not please. But eg: Make sure that Unitrends isn’t exposed.

File: csirt.divd.nl/cases/DIVD-2021-00014.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+1745) directly to the internet until Kaseya has patched these vulnerabilities.
+ 
+## What we are doing
+The Dutch Institute for Vulnerability Disclosure (DIVD) performs a daily scan to detect vulnerable Kaseya Unitrends servers and notify the owners directly or via the known abuse channels, Gov-CERTs and CSIRTs, and other trusted channels.
```
Comment (MrSeccubus) [inline csirt.divd.nl/cases/DIVD-2021-00014.md:38]: Not daily, but frequent. We may miss a day.

File: csirt.divd.nl/cases/DIVD-2021-00014.md [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+
+## Timeline
+
+| Date  | Description |
+|:-----:|-------------|
+| 02-07-2021 | Vulnerabilities discovered. |
```
Comment (MrSeccubus) [inline csirt.divd.nl/cases/DIVD-2021-00014.md:44]: Suggestion: 2 Juli 2021 to avoid confusion.

**Review by MrSeccubus — CHANGES_REQUESTED** [review]

File: csirt.divd.nl/cases/DIVD-2021-00014.md (hunk 2 above)
Comment (cookiemonster) [reply csirt.divd.nl/cases/DIVD-2021-00014.md:5]: To the internet. Also, water is not advised.

File: csirt.divd.nl/cases/DIVD-2021-00014.md (hunk 4 above)
Comment (cookiemonster) [reply csirt.divd.nl/cases/DIVD-2021-00014.md:21]: There is no patch available.

File: csirt.divd.nl/cases/DIVD-2021-00014.md (hunk 5 above)
Comment (cookiemonster) [reply csirt.divd.nl/cases/DIVD-2021-00014.md:34]: Politeness never hurt.  Please make edits where you like. :-)

File: csirt.divd.nl/cases/DIVD-2021-00014.md (hunk 6 above)
Comment (cookiemonster) [reply csirt.divd.nl/cases/DIVD-2021-00014.md:38]: No. This one we scan every day.

File: csirt.divd.nl/cases/DIVD-2021-00014.md (hunk 7 above)
Comment (cookiemonster) [reply csirt.divd.nl/cases/DIVD-2021-00014.md:44]: Ok.

File: csirt.divd.nl/cases/DIVD-2021-00014.md (hunk 2 above)
Comment (MrSeccubus) [reply csirt.divd.nl/cases/DIVD-2021-00014.md]: Nobody likes a s….

**Review by MrSeccubus — APPROVED** [review]

**Comment by MrSeccubus** [comment]
Using admin to merge.

**Pull request merged**